src/blocks/Form/__stories__/Form.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import {Meta, StoryFn} from '@storybook/react';
+
+import {PageConstructor} from '../../../containers/PageConstructor';
+import {FormBlockDirection, FormBlockModel} from '../../../models';
+import FormBlock from '../Form';
+
+import data from './data.json';
+
+export default {
+    title: 'Blocks/Form',
+    component: FormBlock,
+    args: data.default,
+    argTypes: {
+        type: {control: false},
+        direction: {options: FormBlockDirection, control: {type: 'select'}},
+    },
+} as Meta;
+
+const DefaultTemplate: StoryFn<FormBlockModel> = (args) => (
+    <PageConstructor
+        content={{
+            blocks: [
+                {
+                    ...args,
+                    formData: {yandex: data.formData.yandex}, //yandex form
+                },
+                {
+                    ...args,
+                    formData: {hubspot: data.formData.hubspot}, //hubspot form
+                },
+            ],
+        }}
+    />
+);
+
+export const Default = DefaultTemplate.bind({});
+export const ContentForm = DefaultTemplate.bind({});
```

Comment:
Content
```suggestion
export const Content Direction = DefaultTemplate.bind({});
```

Lets add here the `form-content`